tests/Commands/repeater.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { Cli, Api } from '../Setup';
+import { URL } from 'url';
+import { ChildProcess } from 'child_process';
+
+const config = {
+  cmd: process.env['E2E_CLI_CMD'],
+  cluster: process.env['E2E_CLUSTER'],
+  runId: process.env['E2E_RUN_ID'],
+  apiKey: process.env['E2E_REPEATER_API_KEY'],
+  targetUrl: process.env['E2E_REPEATER_TARGET_URL'],
+  maxTestTimeout:
+    parseInt(process.env['E2E_REPEATER_MAX_TEST_TIMEOUT'], 10) * 1000
+};
+
+describe('Repeater Command', () => {
+  const name = `E2E: Repeater Bright CLI (${config.runId})`;
+  const [cmd, ...args]: string[] = config.cmd.split(' ');
+  const cli = new Cli(cmd, args);
+  const targetHost = new URL(config.targetUrl).host;
+  const api = new Api({
+    baseUrl: `https://${config.cluster}`,
+    apiKey: config.apiKey
+  });
+
+  let repeaterId: string;
+  let commandProcess: ChildProcess;
+
+  beforeEach(async () => {
+    repeaterId = await api.createRepeater(name);
+  }, 10000);
+
+  afterEach(async () => {
+    if (commandProcess) {
+      commandProcess.stdin.destroy();
+      commandProcess.stderr.destroy();
+      commandProcess.stdout.destroy();
+      commandProcess.kill();
```

Comment:
Terminate the repeater gracefully using SIGINT or SIGTERM signals
```suggestion
      commandProcess.kill('SIGINT');
```